### ISSUE TITLE
slide to close not working.

### ISSUE THREAD

**Issue by Dramex** [balloob/react-sidebar#169]
Hi, 

in the demo i can slide the sidebar with my finger to close it. but i can do that in my code.
both overlay click / close function works fine.

what the problem here.

**Comment by markusenglund** [comment]
Hi,
The example still uses an old version of the library (2.3.2). I should fix that. In the new version (3.0) touch sliding has been disabled on IOS devices since it forces its own behaviour for horizontal sliding. See #67.

Which iPhone or ipad are you currently using? Do you think swiping right to left should be enabled? I don't have an iPhone on me right now, so I can't check the exact behaviour. It might have been a mistake on my part to remove touch gestures entirely on IOS.

**Comment by Dramex** [comment]
Hi, i'm using iphone x
yes it should be enabled, i actully was doing the sidebar stuff via just CSS, then when i saw this repo i liked the sliding from right to left so much. when it didn't work now sadly i have no difference than before :(

**Comment by Dramex** [comment]
even the sliding from left to right should work in middle of the screen, the bulit in sliding works in the edges

**Comment by Dramex** [comment]
I meant when it closed, there no issues if people slide left to right (in middle of screen) to open it. 
like `Slideout.js` ( https://slideout.js.org )

**Comment by Dramex** [comment]
`sliding from the middle of the screen when sidebar is opened` works fine with `react-sidebar@2` 
there no need to changes anything here.

**Comment by markusenglund** [comment]
I don't follow. If someone wants to open the sidebar with a touch gesture, they have to slide from the edge to the middle right? And that will in turn cause the go-back-in-history behavior on IOS which makes it basically unusable.

**Comment by Dramex** [comment]
@markusenglund Yes, but why from edges to middle? why not just sliding from left to right in middle of the screen (not from edges). it will works fine like (slideout.js)

**Comment by markusenglund** [comment]
Ok I understand what you mean. Unfortunately I think that would take quite a bit of effort to implement in this library. I simply don't have the time to do it currently so I will not implement it.

**Comment by Dramex** [comment]
@markusenglund your library already make div that above the main content, and already works with the (edges) in version @2, Is it hard to implement that to a nice base that you have?

**Comment by amatissebastien** [comment]
We are using this plugin for a Cordova app, so we won't have the issue of the 'swipe to go back' feature that exists in Safari. 

We could use the older version, but is there anything else that might bite us there? The ideal scenario for us would be an option like iosEnableSwipe that we could pass to force it on, knowing the issue in the browser. We could even dynamically set that for our mobile builds vs our browser builds. 

Either way, if you could please let me know if the best solution is to use the older version :) 

Thanks,  Sebastien 

UPDATE: I tried the older version, and its for sure a mess just swapping it out. The code looks really straight forward, so I could customize if needed, but would of course rather have it be a real feature.
